src/components/Layout/PrivateRoute/PrivateRoute.tsx

```diff
@@ -4,32 +4,33 @@ import { useNavigate } from 'react-router-dom'
 import { useAppDispatch } from '@redux/hooks'
 import { setUser } from '@redux/user'
 import { useGetOrCreateUser } from '@data-provider/query'
+import { useSnackBar } from '@components/SnackBarProvider/SnackBarProvider'
 
 const PrivateRoute = (): JSX.Element => {
   const [isAuthorized, setIsAuthorized] = React.useState<boolean>(true)
   const navigate = useNavigate()
   const dispatch = useAppDispatch()
+  const { showSnackBar } = useSnackBar()
 
   const { data, isLoading, error } = useGetOrCreateUser()
 
   React.useEffect(() => {

```

Comment:
It would be better to import only useEffect from react instead of importing the whole library.